src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,20 @@
+import React from 'react';
+import './Todo.css';
+import Todo from './Todo.js';
+
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+
+const TodoList = (props) => {
+    return (
+        <div>
+            {props.array.map(todo => {
+                return (
+                    <Todo key={todo.id} task={todo.task}/>
```

Comment:
what might the trade-offs be for passing down the `todo` object 🆚 passing down `todo.task`?